index.html

```diff
@@ -51,5 +51,152 @@ <h4>I'm Azeezat</h4>
     </ul>
     <div id="background"></div>
     </section>
+    <div class="work-section">
+    <h1>My Recent work</h1>
+    </div>
+
+    <hr class="solid"/>
+
+    <section class="picture">
+    <div class="pictures"></div>
+    <p id="post">Multiple-Post Stories</p>
+    <p class="daily">
+        A daily selection of privately Personalized Reads; no account or signup
+        required has been the industry standard dummy text ever since 1500s,
+        when an unknown printer took a dummy text.
+    </p> 
+    <div class="grid">
+    <ul class="grid-container">
+        <li class="grid-item">Css</li>
+        <li class="grid-item">Html</li>
+        <li class="grid-item">bootstrap</li>
+        <li class="grid-item">Ruby</li>
+    </ul>
+    </div>
+    <button id="button" type="submit">See Project</button>
+
+    <div class="bottom">
+    <a href="#"
+        ><img src="image.png" alt=""><div class="centered">
+        <h3>Professional Art <br>printing Data</h3>A daily selection of privately Personalized Reads; no account or signup
+        required has been the industry standard.<br><ul class="grid-containers"><li class="display">Html</li>
+        <li class="display">bootstrap</li>
+        <li class="display">Ruby</li></ul></div></a><button class="btn active" type="submit">
+        See Project</button></div>
+    <div class="bottom">
+    <a href="#"
+        ><img src="image.png" alt=""> <div class="centered">
+        <h3>Professional Art <br>printing Data</h3>A daily selection of privately Personalized Reads; no account or signup
+        required has been the industry standard.<br><ul class="grid-containers"> <li class="display">Html</li>
+        <li class="display">bootstrap</li>
+        <li class="display">Ruby</li></ul></div></a><button class="btn" type="submit">
+        See Project</button>
+    </div>
```

Comment:
- [ ] [OPTIONAL] In order to make your code more readable please indent your code correctly, this will make it a lot easier to adjust your code in the future.